problems/4.arrays.js

```diff
@@ -2,28 +2,32 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr) {
+  return (arr[0] + arr.length -1) / 2
+  
```

Comment:
YOu're finding the average between the first element and the last INDEX. You need the last element. arr[arr.length - 1]